dockerfiles/theia-endpoint-runtime-binary/src/entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+set -e
+set -x
+
+# Ensure $HOME exists when starting
+if [ ! -d "${HOME}" ]; then
+    mkdir -p "${HOME}"
```

Comment:
Do we always have permissions to create home directory?

Reply:
Yes, but we already use this approach in the devfile registry images: https://github.com/eclipse/che-devfile-registry/blob/master/arbitrary-users-patch/entrypoint.sh#L4

Reply:
We do directly ensure group write permissions to the `/home` directory in the arbitrary patch [dockerfile](https://github.com/eclipse/che-devfile-registry/blob/d38c074eb895d20987c8e5820ea14b951bda0af3/arbitrary-users-patch/Dockerfile#L6)

On OpenShift, the requirement is that any user-writable files are owned by the root group and have group read/write/execute permissions.